lib/pages/profile_page/follows_page/view_follows.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:linear/constants/themeSettings.dart';
+import 'package:linear/model/user.dart';
+import 'package:linear/util/apis.dart';
+import 'package:linear/util/cognito/user_provider.dart';
+import 'package:linear/util/cognito/user.dart' as cognito_user;
+import 'package:linear/pages/profile_page/follows_page/followers_list.dart';
+import 'package:linear/pages/profile_page/follows_page/following_list.dart';
+
+enum _Tab {followers, following}
+
+// ignore: must_be_immutable
+class ViewFollowsPage extends StatefulWidget {
+  const ViewFollowsPage(
+      {super.key, required this.token, required this.username, required this.user, required this.type});
+  final String token;
+  final String username;
+  final User user;
+  final String type;
+
+
+  @override
+  State<ViewFollowsPage> createState() => _ViewFollowsPageState();
+}
+
+class _ViewFollowsPageState extends State<ViewFollowsPage> {
+  cognito_user.User? currentUser = UserProvider().user;
+
+ 
+   _Tab _selectedTab = _Tab.followers;
+   int timesCalled = 0;
```

Comment:
I think this way is too hacky and we can find a simpler way of doing this; But since this works, we'll merge it and enhance it in a future change